database/migrations/2016_10_20_071655_added_by_user_id_column_to_payments_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddedByUserIdColumnToPaymentsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('payments', function (Blueprint $table) {
+            $table->integer('addedByUserId')->nullable();
```

Comment:
Ked sa do DB pridava tato referencia, a su tam uz vyplnene zaznamy bude to padat na tom ze pri tych zaznamoch nie je ziadne id. Cize je potrebne medzi dvojicou tych riadkov tam priradit nejake id.

Cize nieco taketo:

```
$table->integer('addedByUserId')->unsigned();
DB::table('payments')->update(['addedByUserId' => 1]);
$table->foreign('addedByUserId')->references('id')->on('users');
```

by malo fungovat